src/tools/contract.cpp

```diff
@@ -49,7 +49,7 @@ return_code parseArguments(int argc,
         "Number of threads to use")(
         "core,k",
         boost::program_options::value<double>(&contractor_config.core_factor)->default_value(1.0),
-        "Percentage of the graph (in vertices) to contract [0..1]")(
+        "Percentage of the graph (in vertices) to contract [0..1]. Will always be 1.0")(
```

Comment:
would  be `DEPRECATED Percentage of the graph (in vertices) to contract [0..1]` better?

Reply:
done in https://github.com/Project-OSRM/osrm-backend/pull/4601/files#diff-097d6382cc3c7256f51a139534b8c76dR52